Dockerfile_pg_dev

```diff
@@ -1,5 +1,5 @@
 # start with a base image
-FROM postgres:12.5
+FROM postgres:11
```

Comment:
Nope ... this may have fixed your issue but it'll break all ours. Upgrade your local postgres to the expected version instead. (soon to be version 13)